### ISSUE TITLE
Deprecated constant REQUEST_TIME not caught

### ISSUE THREAD

**Issue by vaish** [mglaman/phpstan-drupal#139]
`PHPStan\Rules\Deprecations\AccessDeprecatedConstant` contains hard-coded list of deprecated Drupal constants. It appears that this list needs to be updated.
Here is the up to date list: https://api.drupal.org/api/drupal/deprecated/8.9.x?order=object_type&sort=asc&page=5

And here is the diff between the list used by phpstan-drupal and the current one taken from the above link:

```diff
--- deprecated-constants-phpstan-drupal.txt    2020-08-15 21:47:04.172183984 +0200
+++ deprecated-constants-current.txt    2020-08-15 21:55:03.686038859 +0200
@@ -12,0 +13 @@
+'DRUPAL_PHP_FUNCTION_PATTERN' => 'Deprecated in drupal:8.8.0 and is removed from drupal:9.0.0. Use \Drupal\Core\Extension\ExtensionDiscovery::PHP_FUNCTION_PATTERN instead.',
@@ -29 +30 @@
-'MENU_MAX_MENU_NAME_LENGTH_UI' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\Core\Config\Entity\ConfigEntityStorage::MAX_ID_LENGTH because the menu name is a configuration entity ID.',
+'MENU_MAX_MENU_NAME_LENGTH_UI' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\system\MenuStorage::MAX_ID_LENGTH instead.',
@@ -36,2 +37 @@
-'REGIONS_ALL' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\block\BlockRepositoryInterface::REGIONS_ALL instead.',
-'REGIONS_VISIBLE' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\block\BlockRepositoryInterface::REGIONS_VISIBLE instead.',
+'REQUEST_TIME' => 'Deprecated in drupal:8.3.0 and is removed from drupal:10.0.0. Use \Drupal::time()->getRequestTime().',
@@ -46 +46 @@
-'UPDATE_NOT_CURRENT' => 'Deprecated in in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\update\UpdateManagerInterface::NOT_CURRENT instead.',
+'UPDATE_NOT_CURRENT' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\update\UpdateManagerInterface::NOT_CURRENT instead.',
```
